manifests/3/3CX/3CXSoftphone/16.3.0.220/3CX.3CXSoftphone.locale.en-US.yaml

```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: 3CX.3CXSoftphone
+PackageVersion: 16.3.0.220
+PackageLocale: en-US
+Publisher: 3CX
+PublisherUrl: https://www.3cx.com/
+PublisherSupportUrl: https://www.3cx.com/support/
+PrivacyUrl: https://www.3cx.com/company/privacy/
+#Author: 
+PackageName: 3CX Softphone
+PackageUrl: https://www.3cx.com/
+License: Copyright (c) 3CX
+#LicenseUrl: 
+#Copyright: 
+#CopyrightUrl: 
+ShortDescription: 3CX Softphone
+Description: 3CX softphone for Windows
```

Comment:
```suggestion
Description: Looking for a way to make and receive calls from your PC or laptop? Look no further! The 3CX softphone for Windows is a free app developed by 3CX, for use independent of the phone system. Once connected to your VoIP provider, it can be used to make and receive calls over your internet connection.
```